dev-util/desktop-file-utils/files/desktop-file-utils-0.26-support-version-1.5.patch

```diff
@@ -0,0 +1,83 @@
+From 425177a28b6215e0745f95100160a08e810fd47c Mon Sep 17 00:00:00 2001
```

Comment:
Nit: include a link to the commit at the top?

Reply:
Added.